docs/ref/settings.txt

```diff
@@ -1122,6 +1122,16 @@ Default exception reporter filter class to be used if none has been assigned to
 the :class:`~django.http.HttpRequest` instance yet.
 See :ref:`Filtering error reports<filtering-error-reports>`.
 
+.. setting:: DEFAULT_EXTRA_HEADERS
+
+``DEFAULT_EXTRA_HEADERS``
+-------------------------
+
+Default: ``{}``
+
+Default extra headers to use for various automated email correspondence from the
+site manager(s).
```

Comment:
That doesn't seem technically correct. Won't the default headers exist on _any_ email your django project generates?